src/main/java/com/blakebr0/extendedcrafting/tile/TileCraftingCore.java

```diff
@@ -196,15 +198,15 @@ public int getPedestalCount() {
 	public void readFromNBT(NBTTagCompound tag) {
 		super.readFromNBT(tag);
 		this.inventory.deserializeNBT(tag);
-		this.progress = tag.getInteger("Progress");
+		this.progress = tag.getLong("Progress");
```

Comment:
Is this is safe w.r.t. loading old worlds?

Reply:
(I'm guessing yes because of 41d665ce0c814fcb754382de93ea39c6c76f2ab8)

Reply:
correct - when running `NBTTagInt#getLong()` it returns `(long)this.data`, and `NBTTagCompound#getLong()`  only cares if the NBT entry extends `NBTPrimitive`, which both `NBTTagInt` and `NBTTagLong` do.